core/src/main/java/org/bitcoinj/core/TransactionInput.java

```diff
@@ -115,40 +115,50 @@ public static TransactionInput read(ByteBuffer payload, Transaction parentTransa
         TransactionOutPoint outpoint = TransactionOutPoint.read(payload);
         byte[] scriptBytes = Buffers.readLengthPrefixedBytes(payload);
         long sequence = ByteUtils.readUint32(payload);
-        return new TransactionInput(parentTransaction, scriptBytes, outpoint, sequence, null);
+        return new TransactionInput(parentTransaction, null, scriptBytes, outpoint, sequence, null, null);
```

Comment:
I would consider using the 5-argument "internal use only" constructor in these 3 places.

Reply:
Changed in https://github.com/bitcoinj/bitcoinj/compare/542f34cf7fa52ba065aa0e4650c3bddc6aba0e12..0b597887e953c7445e2ea0ab6c750143f30d882d, but what are the other two places?